### DISCUSSION TITLE
Customer Management Framework

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#16707] [Q&A / Support]
Can I get any document or video to understand "Customer Management Framework" - in pimcore default document, it's very difficult  to understand, and don't know how to use the feature of "Customer Management Framework"  as a developer

**Comment by patriciomg** [comment]
Hi,
there are no videos on YT with the current version, well I haven't found them yet :D 
but you can find a video tutorial in the Pimcore Academy, but for the academy you need to buy a subscription.
cheers,
P.

**Comment by idna001** [comment]
Hey, 

the documentation is a bit rare. But it shows [how to install](https://pimcore.com/docs/platform/Customer_Management_Framework/Installation/) the CMF on a pimcore.
You can also try the functionalities in the demo: 
![grafik](https://github.com/pimcore/pimcore/assets/46223359/b7ed3a97-a641-40fe-81fc-5d36839fbd0c)
The data is coming from the registrations in the frontend. So there, you can look up the API to the framework. 
Or maybe you want the same user data as in the demo shown, then you can just export the class from the demo. 

Hope that helps
Andreas
